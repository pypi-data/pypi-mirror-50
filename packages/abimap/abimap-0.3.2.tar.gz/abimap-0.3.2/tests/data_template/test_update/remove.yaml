# Testing remove strategy
-
  input:
    args:
      - "update"
      - "--remove"
      - "--allow-abi-break"
      - "remove.map"
    stdin: "symbol.in"
  output:
    file:
    stdout: "remove_sanity.stdout"
    warnings:
      - "ABI break detected: symbols were removed."
    errors:
    exceptions:
-
  input:
    args:
      - "update"
      - "--remove"
      - "--final"
      - "--allow-abi-break"
      - "remove.map"
    stdin: "symbol.in"
  output:
    file:
    stdout: "remove_final.stdout"
    warnings:
      - "ABI break detected: symbols were removed."
    errors:
    exceptions:
-
  input:
    args:
      - "update"
      - "--remove"
      - "--allow-abi-break"
      - "--out"
      - "remove.map"
      - "remove.map"
    stdin: "symbol.in"
  output:
    file: "remove.out"
    stdout: "remove.stdout"
    warnings:
      - "Overwriting existing file 'remove.map'"
      - "Given paths in '--out' and 'file' are the same."
      - "Moving 'remove.map' to 'remove.map.old'."
      - "ABI break detected: symbols were removed."
    errors:
    exceptions:
-
  input:
    args:
      - "update"
      - "--remove"
      - "--allow-abi-break"
      - "unexistent_symbol.map"
    stdin: "symbol.in"
  output:
    file:
    stdout: "unexistent_symbol.stdout"
    warnings:
      - "Requested to remove 'symbol', but not found."
    errors:
    exceptions:
-
  input:
    args:
      - "update"
      - "--remove"
      - "--allow-abi-break"
      - "remove_with_wildcard.map"
    stdin: "symbol.in"
  output:
    file:
    stdout: "remove_with_wildcard.stdout"
    warnings:
      - "WITH_WILDCARD_1_0_0 contains the '*' wildcard in global scope. \
        It is probably exporting symbols it should not."
      - "The '*' wildcard was found in more than one place:"
      - "WITH_WILDCARD_1_0_0: in 'global'"
      - "WITH_WILDCARD_1_0_0: in 'local'"
      - "ABI break detected: symbols were removed."
      - "Wildcard '*' found in global. Removed to avoid exporting unexpected \
        symbols."
    errors:
    exceptions:
-
  input:
    args:
      - "update"
      - "--remove"
      - "base.map"
    stdin: "remove.in"
  output:
    file:
    stdout: "dont_break_api.stdout"
    warnings:
    errors:
      - "ABI break detected: symbols would be removed"
    exceptions:
      - "ABI break detected: symbols would be removed"
