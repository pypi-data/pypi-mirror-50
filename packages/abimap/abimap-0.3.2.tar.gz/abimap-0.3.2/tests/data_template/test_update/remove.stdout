Removed:
    symbol

Merging all symbols in a single new release
